docs/kyma/docs/017-details-local-installation.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Local installation
+type: Details
+---
+
+For better understanding of complex installation process, this guide describes it step by step.
+
+## Start local installation
+
+To fire local installation run following command:
+```
+./installation/cmd/run.sh
+```
+
+This script sets up default parameters, starts Minikube, builds Kyma-Installer, generates local configuration, creates `installer` Custom Resource and sets up Installer. Subsequent sections provide a detailed description of each step.
```

Comment:
- Kyma-Installer -> and not `the Kyma installer` like here: https://github.com/kyma-project/kyma/blob/96313927b69dda5921a6024f390dc697b56eddf8/installation/README.md ?
- creates `installer` Custom Resource and sets up Installer. -> ``creates the `installation` Custom Resource, and sets up the Installer. ``  -  isn't the CR called `installation` like here: https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/040-cr-installation.md ?

Reply:
changed

Reply:
changed